### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by Chaos02**

### CODE REVIEW

File: custom_components/intex_spa/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		"cannot_connect": "Verbindungsfehler",
+		"dns_not_known": "Fehler beim Aufl�sen der DNS-Adresse",
+		"unknown": "Unerwarteter Fehler"
+	 },
+	 "step": {
+		"user": {
+		  "description": "Wenn sie Hilfe bei der Einrichtung ben�tigen schauen sie hier: https://github.com/mathieu-mp/homeassistant-intex-spa",
+		  "data": {
+			 "name": "Ger�t Name",
+			 "host": "Ger�t-Adresse (IP)"
```
Comment (mathieu-mp) [inline custom_components/intex_spa/translations/de.json:16]: This can either be an IP or a host[.domain], please remove the mention of the IP

**Review by mathieu-mp — CHANGES_REQUESTED** [review]
Thank you for providing this MR !

Both files has to be encoded as UTF-8, can you make this change please ?

File: custom_components/intex_spa/translations/de.json (hunk 1 above)
Comment (Chaos02) [reply custom_components/intex_spa/translations/de.json:16]: of course I can change it to UTF-8! oops...

regarding the mention of the IP - I found it a little obstruse and most people have the device on an internal IP. 
Maybe there is a better way of saying "fully qualified domain adress". (most people do not know what it means and have to look it up first)
I will try to come up with something better!

**Review by mathieu-mp — APPROVED** [review]

**Comment by mathieu-mp** [comment]
Nice job, thank you !

**Pull request merged**